backend/src/auth/auth.controller.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  Post,
+  Body,
+  Patch,
+  Controller,
+  Get,
+  UseGuards,
+  Req,
+  Res,
+  HttpStatus,
+} from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { FortyTwoAuthGuard } from './42-auth/42.guards';
+import { JwtAuthGuard } from './jwt-auth/jwt-auth.guard';
+import { Response } from 'express';
+//import { UserService } from '../user/service/user.service'
+import { User } from '../user/domain/model/user/user';
+import { UserService } from '../user/service/user.service';
+//teste deletar
+
+const teste = new User({
```

Comment:
After doing all your tests, you need to remove all unnecessary code to ensure the code quality